inc/Engine/Optimization/RUCSS/Controller/UsedCSS.php

```diff
@@ -25,6 +25,13 @@ class UsedCSS {
 	 */
 	private $purge;
 
+	/**
+	 * Status of CPCSS option.
+	 *
+	 * @var bool
+	 */
+	private $cpcss_enabled = false;
```

Comment:
What if we will inject either Options or directly Options->cpccss_enable in the contructor and remove the set method?